app/catalog_data.go

```diff
@@ -27,6 +27,11 @@ const (
 	systemLibraryBranch = "master"
 	systemLibraryName   = "system-library"
 	defSystemChartVer   = "management.cattle.io/default-system-chart-version"
+
+	helm3LibraryURL    = "https://git.rancher.io/helm3-charts"
+	helm3LibraryBranch = "master"
+	helm3LibraryName   = "helm3-library"
+	helm3HelmVersion   = "helm_v3"
```

Comment:
Is there a const in `pkg/catalog/helm` that can be used here? This should be defined in one place to keep it consistent for future changes.